reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -423,9 +424,14 @@ class StaticHtmlReporter : Reporter() {
     }
 
     private fun TBODY.evaluatorRow(error: ResolvableIssue) {
-        val cssClass = when {
-            error.isResolved -> "ort-resolved"
-            else -> "ort-error"
+        val cssClass = if (error.isResolved) {
+            "ort-resolved"
+        } else {
+            when (error.severity) {
+                Severity.ERROR -> "ort-error"
+                Severity.WARNING -> "ort-warning"
+                Severity.HINT -> "ort-warning"
```

Comment:
Is it intended to not throw `NotImplementedException` here now?

Reply:
Yes. In the preceeding variant of this change i threw it only in case severity is not error, warning or hint. Now this case is covered by a compiler error which is even better.

As commit message says warnings and hints now are shown in yellow. 
Adjusting the hint style imo can be done separately and I think its not very important to have it right now.

Reply:
I agree it's not important to have a hint styling right now since we don't even create hints, but using an "ort-warning" style for `Severity.HINT` simply looks like a copy & paste error here.